### PR TITLE
Nikita/API: Code sample page should redirect to Quickstart page

### PR DESCRIPTION
**Pull request by nikita-deriv**

### CODE REVIEW

File: _includes/demos/buy-contract-python.py [hunk 1]
```diff
@@ -1,35 +1,32 @@
-
-# This is an example of using Python to buy a contract via the Deriv/Binary API
-# This is a simple example of buying a contract where we do not check first if
-# the user has the symbol available to buy. To check this you would use the active_symbols call.
+# This is an example of using Python to buy a contract via the Deriv/Binary API.
+# This is a simple example of buying a contract where we do not check first ifthe user has the
```
Comment (khalid-deriv) [inline _includes/demos/buy-contract-python.py:2]: Forgot the space in "ifthe" - `check first if the user`

File: _includes/demos/balance-csharp.cs [hunk 2]
```diff
@@ -66,7 +58,6 @@ public async Task StartListen()
                 do
                 {
                     result = await this.ws.ReceiveAsync(new ArraySegment&lt;byte&gt;(buffer.Array), CancellationToken.None);
```
Comment (khalid-deriv) [inline _includes/demos/balance-csharp.cs:60]: Shouldn't this be `ArraySegment<byte>` ? Looks like it's using generics or is it something else that I don't know? 🤔

File: _includes/demos/balance-csharp.cs [hunk 3]
```diff
@@ -8,53 +8,45 @@
...
         {
-
             while (this.ws.State == WebSocketState.Connecting) { };
             if (this.ws.State != WebSocketState.Open)
             {
                 throw new Exception("Connection is not open.");
             }
-
             var reqAsBytes = Encoding.UTF8.GetBytes(data);
             var ticksRequest = new ArraySegment&lt;byte&gt;(reqAsBytes);
```
Comment (khalid-deriv) [inline _includes/demos/balance-csharp.cs:41]: Shouldn't this be ArraySegment<byte> ? Looks like it's using generics or is it something else that I don't know? 🤔

File: _includes/demos/balance-php.php [hunk 4]
```diff
@@ -1,33 +1,33 @@
 &lt;?php
```
Comment (khalid-deriv) [inline _includes/demos/balance-php.php:1]: Looks like something is wrong with the `<` and `>` signs. Is it intentional?

File: _includes/demos/proposal-csharp.cs [hunk 5]
```diff
@@ -6,64 +6,55 @@
...
             while (this.ws.State == WebSocketState.Connecting) { };
             if (this.ws.State != WebSocketState.Open)
             {
                 throw new Exception("Connection is not open.");
             }
-
             var reqAsBytes = Encoding.UTF8.GetBytes(data);
-            var ticksRequest = new ArraySegment&lt;byte&gt;(reqAsBytes);
-
+            var ticksRequest = new ArraySegment&lt;byte>(reqAsBytes);
```
Comment (khalid-deriv) [inline _includes/demos/proposal-csharp.cs:39]: Hmmm you changed the `>` sign but not the `&lt;`... 🤔

File: _includes/demos/balance-php.php (hunk 4 above)
Comment (nikita-deriv) [reply _includes/demos/balance-php.php:1]: This approach was used for preventing from reading them as tags in code representation.

File: _includes/demos/balance-csharp.cs (hunk 3 above)
Comment (nikita-deriv) [reply _includes/demos/balance-csharp.cs:41]: Explained this in next comment

File: _includes/demos/buy-contract-python.py (hunk 1 above)
Comment (nikita-deriv) [reply _includes/demos/buy-contract-python.py]: Eagle, eye. Replaced.

File: _includes/demos/proposal-csharp.cs (hunk 5 above)
Comment (nikita-deriv) [reply _includes/demos/proposal-csharp.cs]: Thanks for noticing, updated here.

**Review by khalid-deriv — APPROVED** [review]
Looks good to go 👍🏼

**Pull request merged**